install/kubernetes/helm/istio/values.yaml

```diff
@@ -185,6 +195,14 @@ gateways:
     - port: 31400
       name: tcp
       nodePort: 31400
+    # Pilot and Citadel MTLS ports are enabled in gateway - but will only redirect
+    # to pilot/citadel if global.meshExpansion settings are enabled.
```

Comment:
Change comment to say that while ports are exposed by the service, routing will happen only after user adds gateway config and virtual service per tutorial. 
This way we don’t need to tweak helm for ingress based mesh expansion. You can add or remove routing config to enable or disable mesh expansion.